sig-windows/annual-report-2020

```diff
@@ -0,0 +1,85 @@
+[ ] by March 8th, 2021
+[ ] titled: annual-report-YEAR.md
+[ ] are there any responses that you’d like to share privately first? steering-private@kubernetes.io or tag your liaison in for discussion. 
+
+# Special Interest Groups:
+(do not copy if you are responding for a working group)
+
+
+# Instructions for this section:
+[ ] each response should have supporting documentation, linked KEPs, graphs, data
+[ ] err on the side of being descriptive with your responses. End users and others that read this might not know our upstream acronyms, shorthand, or where to find a reference. 
+Operational
+
+How are you doing with operational tasks in sig-governance.md?
+Is your README accurate?have a CONTRIBUTING.md file?
+Yes / Yes - README.md is up to date, CONTRIBUTING.md is still accurate but could use some updates.  
+
+All subprojects correctly mapped and listed in sigs.yaml?
+Yes
+
+What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?
+Meetings have decent attendance but few active participants. 
+Meeting notes are used during the meetings are up to date. 
+Recordings are up to date but few views 
+
+How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?
+Owners files are up to date. Existing subprojects are mostly in maintenance mode. We would like to try starting one or two subprojects to gather more interest / new participation in SIG-Windows.
+Same question as above but for working groups.
+N/A
+
+When was your last monthly community-wide update? (provide link to deck and/or recording)
+Sept 17 Slides recording
+Membership
+
+Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+Yes
+
+How do you measure membership? By mailing list members, OWNERs, or something else?
+We don’t measure membership.
+
+How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?
+We are blocked on reviewer/approver bandwidth from other SIGs much more often than reviewer/approver bandwidth in our SIG.
+Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?
+We want to focus on this in the next few months. Some ideas we have are pairing with folks and establishing one or two new focused subprojects that will encourage new participants.
+
+What programs do you participate in for new contributors? We’re wanting to groom long-term/dedicated contributors. Are there programs to help with this? Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?
```

Comment:
need a line break after the contributors question

Reply:
and then question for long-term/dedicated contributors - to a specific subproject? what kind of contributors? windows experts? can you elaborate here?